src/session/content/chat_history.rs

```diff
@@ -209,7 +211,7 @@ impl ChatHistory {
         if adj.value() < adj.page_size() * 2.0 || adj.upper() <= adj.page_size() * 2.0 {
             if let Some(chat) = self.chat() {
                 spawn(clone!(@weak chat => async move {
-                    chat.history().load_older_messages().await;
+                    let _ = chat.history().load_older_messages(20).await;
```

Comment:
This should also be logged.

Reply:
Yeah, maybe we distinguish between the error variants and just warn in case of an `AlreadyLoading`? Because it's not really a fatal error.

Reply:
I don't think there's a need to log for `AlreadyLoading` in this case, since it's perfectly normal that this function is fired more than once while scrolling (especially if you consider touchpad and touchscreen input).